src/content/eth2/shard-chains/index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Shard chains
+description: Everything you need to know about shard chains
+lang: en
+sidebar: true
+---
+
+# Scaling and shard chains
+
+<Subtitle>Shard chains are Eth2's answer to the scaling problem. They'll turn Ethereum into a super highway of interconnected blockchains. </Subtitle>
+
+## What is a shard chain?
```

Comment:
Might be worth drawing a parallel to sharded databases? Explaining that this draws from decades of innovation & optimizations of database technology may make it seem more familiar / give a frame of reference to some folks vs. making it sound like this is completely new & foreign technology.